components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/device/constants/Constants.java

```diff
@@ -51,11 +51,19 @@
     public static final String SEPARATED_WITH_SPACE = " ";
     public static final String ERROR = "error";
     public static final String ERROR_DESCRIPTION = "error_description";
+    public static final String DEVICE_ENDPOINT_PATH = "/authenticationendpoint/device.do";
+    public static final String DEVICE_SUCCESS_ENDPOINT_PATH = "/authenticationendpoint/device_success.do";
+    public static final String USERCODE_QUANTIFIER_CONSTRAINT = "USRCDE_QNTFR_CONSTRAINT";
+
+    public static final String EXPIRY_TIME_PATH = "OAuth.SupportedGrantTypes.SupportedGrantType.ExpiryTime";
+    public static final String CONF_KEY_SET = "OAuth.SupportedGrantTypes.SupportedGrantType.KeySet";
+    public static final String CONF_USER_CODE_LENGTH = "OAuth.SupportedGrantTypes.SupportedGrantType.UserCodeLength";
+    public static final String RESPNSE_TO_STRING_PATH = "OAuth.SupportedGrantTypes.SupportedGrantType.ResponseToString";
 
     // Configurable values.
     public static final int KEY_LENGTH = 6;
-    public static final long EXPIRES_IN_VALUE = 600000L;
-    public static final int INTERVAL_VALUE = 5000;
+    public static final long EXPIRES_IN_MILLISECONDS = 600000L;
+    public static final int  INTERVAL_MILLISECONDS = 5000;
```

Comment:
milliseconds need to be of long type.

Reply:
As these variables are used in public methods, problems in changing the data types, also this variable is used directly without any changes to the value.